rust/lon/src/cli.rs

```diff
@@ -452,11 +473,20 @@ fn bot_fallible(directory: impl AsRef<Path>, forge: &impl Forge, base_ref: &str)
             .update()
             .with_context(|| format!("Failed to update {name}"))?;
 
-        let Some(summary) = summary else {
+        let Some(mut summary) = summary else {
             log::info!("No updates available");
             continue;
         };
 
+        let mut body: Option<String> = None;
+
+        if detailed {
+            let diff = source.diff(&summary, num_commits)?;
+
+            summary.details = Some(format!("Last {num_commits} commits:\n{diff}"));
+            body = Some(format!("```\n{diff}\n```"));
```

Comment:
The body would miss the "Last XX commits" line, right?

We should probbaly just add a `body()` fn to `CommitMessage` and always use that for the body of the PR. This functions would return everything but the first line of the commit.